src/tiled-map-resource.ts

```diff
@@ -42,6 +42,18 @@ export enum TiledMapFormat {
    JSON = 'JSON'
 }
 
+export interface TiledMapOptions {
+   /**
+    * Optionally override the map format, by default files ending in .tmx are treated as TMX format, otherise treated as JSON format
+    */
+   mapFormatOverride?: TiledMapFormat;
+
+   /**
+    * Optionally Specify the first layer z-index value, each layer will increment up 1 unless overridden (default -1)
+    */
+   firstLayerZIndex?: number;
```

Comment:
See above

```suggestion
   startingLayerAutoIncrementingZIndex?: number;
```

Reply:
This doesn't really influence the auto increment, just the starting z-index. Previously it ways named `layerZIndexStart` which is maybe better?

You suggestion is more descriptive, is there a more terse description? Maybe `initalLayerZIndex`, `startingLayerZIndex`, or go back to the original name?

Reply:
Ah I see! I think either of those are fine, `startingLayerZIndex` is succint and descriptive and `starting` implies it starts a sequence where as `initial` seems static.

Reply:
```suggestion
   startingLayerZIndex?: number;
```